altair/vegalite/v5/api.py

```diff
@@ -132,7 +132,19 @@
 _TSchemaBase = TypeVar("_TSchemaBase", bound=core.SchemaBase)
 _T = TypeVar("_T")
 _OneOrSeq = TypeAliasType("_OneOrSeq", Union[_T, Sequence[_T]], type_params=(_T,))
-"""One of ``_T`` specified type(s), or a `Sequence` of such."""
+"""One of ``_T`` specified type(s), or a `Sequence` of such.
```

Comment:
I'll be moving this into `_typing.py` eventually, but wanted to leave the trail of the origin in [`3fa4a3f` (#3427)](https://github.com/vega/altair/pull/3427/commits/3fa4a3f7baa26e7ea4c6eacac778b91a9801bd58)